config/testdata/conf.good.yml

```diff
@@ -298,6 +298,13 @@ scrape_configs:
   eureka_sd_configs:
     - server: 'http://eureka.example.com:8761/eureka'
 
+- job_name: scaleway
+  scaleway_sd_configs:
+    - role: instance
+      secret_key: 11111111-1111-1111-1111-111111111111
```

Comment:
why is this still working? this is not a good conf as we are missing access key.